tfjs-backend-webgpu/README.md

```diff
@@ -15,8 +15,7 @@ the following models:
 - Speech commands
 
 Note that WebGPU hasn't been officially supported by browsers. Only Google
```

Comment:
This sentence also needs a change.

Reply:
Maybe you can change the whole paragraph to "Google Chrome started to support WebGPU by default in M113 on May 2, 2023".